hooks/post_gen_project.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import contextlib
+import importlib.util
+import os
+import pathlib
+import re
+import subprocess
+
+PROJECT_ROOT = pathlib.Path().resolve()
+NOXFILE_PATH = PROJECT_ROOT / "noxfile.py"
+
+
+def is_django_support_enabled():
+    """
+    Check if DJANGO SUPPORT is enabled.
+    """
+    # Since .cruft.json is not available at this point, we need to check from noxfile.py.
+    spec = importlib.util.spec_from_file_location("noxfile", NOXFILE_PATH)
+    noxfile = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(noxfile)
+
+    return hasattr(noxfile, "DJANGO_VERSIONS")
+
+
+_DJANGO_ONL_FILE_MARKER = re.compile(
+    r"^# cookiecutter-rt-pkg macro: requires cookiecutter.is_django_package$",
+    re.MULTILINE,
+)
```

Comment:
This feels like a beginning of a bespoke cookiecutter language, but I guess you gotta do what you gotta do. :D